storybooks/storybook
# CLI: Install latest storybook testing-library version in init

Closes https://github.com/storybookjs/testing-library/issues/39



**This PR is targeting main**

## What I did

This PR removes a version specifier when adding `@storybook/testing-library`, resolving the referenced issue

## How to test


1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template react-vite/default-ts`
2. Check that you did not get `@storybook/testing-library@0.0.14-next.2` as a dependency
3. Check that you did get `@storybook/testing-library@0.2.0` as a dependency



## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
